### PR TITLE
add accurate page description

### PR DESCRIPTION
**Pull request by JorgeEdPerezGa**

### CODE REVIEW

**Comment by jdursema** [comment]
looks good!

File: src/components/Breath/index.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ const Breath = () => {
   return (
     <section className = 'journal'>
       <h1 className = 'sub-logo'>Breath</h1>
-      <p className = 'description'>Slow down your breath by following the tead dot eader will be distracted by the readable content of a page when looking at its layout. Remember of a page when looking at its layout. </p>
```
Comment (Cache123) [inline src/components/Breath/index.js]: I like the description revamp! Well worded and informative

File: src/containers/Register/index.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ export class Register extends Component {
       <section className = 'register'>
         <article className = 'page-header'>
           <h1 className = 'sub-logo'>Register</h1>
```
Comment (Cache123) [inline src/containers/Register/index.js]: Not sure what you were saying to the user before but the next one is more informative.

File: src/containers/Support/index.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ export class Support extends Component {
       <section className = 'support'>
         <article className = 'support-header'>
           <h1 className = 'sub-logo'>We Are Not Alone</h1>
-          <p className = 'support-description'>Keeping track of the things we like It is a long established fact.</p>
```
Comment (Cache123) [inline src/containers/Support/index.js]: Meaningful support descriptions are great!

**Pull request merged**